build/ios/.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
 ios-dependencies.tgz
 ios-dependencies.tgz.asc
+*.tgz
+*.tgz.asc
```

Comment:
Note to self: the rule above becomes redundant, then. Also, it should probably make sense to add a similar rule also for Android, which is currently missing it.